aces/Loris
# [Tools] Add command line helper class

## Brief summary of changes

In the PR for the raisinbread refresh, @ridz1208 and I thought it would be a good idea to create a "Command Line Helper" class that PHP tool scripts can optionally use to help with common tasks.

I'll keep this file simple to help with code review. We can iterate on the functions of the class as we go forward.

#### Testing Instructions

Right now the class provides helper functions to print colourful output as well as an option to log the output of those functions to a file in `tools/logs/` using the same naming format as the install script.

Copy the following code into a new PHP file called `test.php` in `tools/`.

```php
<?php

require_once 'generic_includes.php';

$helper = new OutputWrapper();
$helper->enableLogging($argv[0]);
$helper->printSuccess('The class is working!');
```

1. Verify that the message prints in green text
2. Check `tools/logs/` and make sure a new log file was created with the output.
